integrations/phone/examples/common-interactions/README.md

```diff
@@ -0,0 +1,61 @@
+- For user input timeouts to be handled as digressions, go to your `Actions Settings > Change Conversation Topic > Conversation Topic` and enable `Allow changing the conversation topic for all actions.`
+- Go to the `Actions > All items > Created by you` and you will see a `Phone Utilities` folder with a list of actions that can be used for phone interactions.
+
+<img src="./assets/phone-utilities-actions.png" />
+
+### Using the phone channel
+
+Once you've setup the starter kit, if you haven't provisioned a phone integration, you can deploy one following [these steps](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-deploy-phone).
+
+Here are some example phrases you can do to trigger the different phone utilities while on the phone with your Watson Assistant:
+
+| Utterance | Phone Behavior | Action Title |
+| --- | --- | --- |
+| `Goodbye` or `You've answered all of my questions` | Watson Assistant will disconnect the call. | `Call Completion` |
+| `<<< SILENCE >>>` | Watson Assistant will reprompt on the phone. | `vgwPostResponseTimeout` |
+| `can I speak to an agent` | Watson Assistant will attempt to transfer your call. | `Connect To Agent` |
+
+
+
+### Starter Kit Details
+#### Transferring the call via the SIP Transfer option in the Connect To Agent Resolver
+
+For phone interactions, typically a SIP transfer is required for transferring the call to an upstream SIP provider. In the Watson Assistant actions, you can use the SIP Transfer Option in the Connect To Agent ressolver to achieve this. 
+
+1. Go to the "Connect To Agent" action, click on the first Action Step and click on `Edit Settings` under the `Connect to Agent Resolver`
+
+<img src="./assets/connect-to-agent-resolver.png" />
+
+2. To add your SIP Transfer information you can edit the fields such as `uri` and add other SIP headers in the configuration options of the Connect To Agent Resolver:
+
+<img src="./assets/sip-transfer-option.png" />
+
+You can find  more details on call transfers on the [Transferring a call to a human agent](https://cloud.ibm.com/docs/assistant?topic=assistant-dialog-voice-actions#dialog-voice-actions-transfer) documentation.
+
+
+#### Speech Recognition Settings
+
+For phone interactions, there may be cases depending on the use case where you need to tune the speech recognition parameters an example of this can be seen in the `Greeting` action (`Actions > Set by Assistant > Greeting`), specifying a `speech_to_text` response type allows you to modify the Speech recognition settings for the interactions with Watson Assistant over the phone:
+
+<img src="./assets/greeting-speech-to-text.png"/>
+
+For more information on modifying the Speech recognition settings, you can find the documentation in the [docs for the service](https://cloud.ibm.com/docs/assistant?topic=assistant-dialog-voice-actions#dialog-voice-actions-speech-advanced)
```

Comment:
Rewrite: For more information on modifying speech recognition settings, see [Applying advanced settings to the Speech to Text service](link).

Reply:
```suggestion
For more information on modifying speech recognition settings, see [Applying advanced settings to the Speech to Text service](https://cloud.ibm.com/docs/assistant?topic=assistant-dialog-voice-actions#dialog-voice-actions-speech-advanced). 
```